### ISSUE TITLE
Cannot read property '_canvasDataGridUniqueId' of undefined

### ISSUE THREAD

**Issue by meixuexiang** [TonyGermaneri/canvas-datagrid#90]
```
  datagridOpts: CanvasDatagridOptions = {
    parentNode: null,
    editable: false,
    selectionMode: 'row',
    autoResizeColumns: true
  };

grid.data = new Array(10).fill([]);

```

> PayslipComponent.html:3 ERROR TypeError: Cannot read property '_canvasDataGridUniqueId' of undefined
    at Object.self.scroll (events.js:269)
    at Object.self.resize (events.js:244)
    at HTMLElement.schemaSetter (intf.js:974)
    at CanvasDatagridComponent.ngOnChanges (canvas-datagrid.component.ts:36)
    at checkAndUpdateDirectiveInline (core.js:10291)
    at checkAndUpdateNodeInline (core.js:11818)
    at checkAndUpdateNode (core.js:11761)
    at debugCheckAndUpdateNode (core.js:12640)
    at debugCheckDirectivesFn (core.js:12585)
    at Object.View_PayslipComponent_0._co [as updateDirectives] (PayslipComponent.html:6)

**Comment by TonyGermaneri** [comment]
Thanks, this looks like a bug that was introduced a few days ago in v0.19.0.  I'll get it patched up ASAP.

**Comment by TonyGermaneri** [comment]
OK, I think I've fixed it in v0.19.3.  I'll put the ticket in QA for a few days.

**Comment by TonyGermaneri** [comment]
This appears to have been fixed.  Moving it into the done column.  Please reopen the ticket if you see the error reoccur.
